src/constants.js

```diff
@@ -102,4 +102,5 @@ export default {
         INSTANT_LAUNCH_LISTING: "listing",
     },
     VICE_LOGS_POLLING_INTERVAL: 10000,
+    DATA_STORAGE_QUOTA_LIMIT: 107374182400, //100 GiB in bytes
```

Comment:
In future this will come from QMS. For external deployments, if QMS is not deployed, this will be default value.